### PR TITLE
kube-proxy: fix the panic caused by uninitialized IPVS interface in ipvs mode

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
kube-proxy: fix the panic caused by uninitialized IPVS interface in ipvs mode

**Which issue(s) this PR fixes**:

Ref #89729

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
kube-proxy: fix the panic caused by uninitialized IPVS interface in ipvs mode
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

File: cmd/kube-proxy/app/server_others.go [hunk 1]
```diff
@@ -257,6 +257,10 @@ func newProxyServer(
 		proxymetrics.RegisterMetrics()
 	} else if proxyMode == proxyModeIPVS {
 		klog.V(0).Info("Using ipvs Proxier.")
+		// fast-fail if ipvsInterface is nil
+		if ipvsInterface == nil {
+			return nil, fmt.Errorf("unable to initialize IPVS interface")
```
Comment (liggitt) [inline cmd/kube-proxy/app/server_others.go:262]: so this would still crash on first run, it would just do so with an error instead of a panic?

Comment (SataQiu) [reply cmd/kube-proxy/app/server_others.go:262]: Yeah, but this doesn't happen on every first run.
It's probably rare, for some wired reason (about kernel?)
Anyway, we should check the nil pointer first.

**Comment by SataQiu** [comment]
/close 
by #89823

**Pull request closed without merging**